gitextensions/gitextensions
# Add a dropdown to enable tag sign

Fixes #3842  .

Changes proposed in this pull request:
 - New checkbox into "Create tag" dialog
 - git tag command that depends on this checkbox
 
 
Screenshots before 

![image](https://user-images.githubusercontent.com/842026/28122167-ab427f52-671d-11e7-955e-d813b09b26fa.png)



Screenshot after 
![image](https://user-images.githubusercontent.com/842026/28122147-9cb933b8-671d-11e7-826f-bba239c4ad6a.png)




How did I test this code:
 - Open "Create tag" dialog
 - Create an annotated tag as usual
 - Verify that the tag is not signed `git tag -v <tagname>`
 - Open "Create tag" dialog again
 - Create a tag with "signed" checkbox enabled
 - Verify that the tag is signed


Has been tested on (remove any that don't apply):
 - GIT 2.11 and above
 - Windows 7 and above
- GitExtension 2.50
